sources/platform/runtime/machine-image/ami-v662.md

```diff
@@ -0,0 +1,217 @@
+page_main_title: v6.6.2
+main_section: Platform
+sub_section: Runtime
+sub_sub_section: Machine image
+page_title: Description of what is available in Machine Image v6.6.2
+page_description: A complete list of language versions, Docker versions, packages and tools available in Machine Image v6.6.2
+page_keywords: CI/CD, shippable CI/CD, documentation, shippable, config, yml, AMI, Docker
+
+# Machine image v6.6.2 (Docker TAG v6.6.2)
+
+**Release Date: june 28, 2018**
+
+**What is installed on the machine image?**
+
+## u14 sku
+* Host Operating System: Ubuntu 14.04 LTS
+* Build Container Operating System: Ubuntu 16.04.3 LTS
+* Kernel Version: 3.13.0-125-generic
+* **Docker Server Version: 17.06.0-ce**
+* Storage Driver: aufs
+* Root Dir: /data/aufs
+* Backing Filesystem: extfs
+* Dirperm1 Supported: false
+* Cgroup Driver: cgroupfs
+* Shippable Official u14 and u16 Docker Images with TAG: `v6.6.2`
+
+## u16 sku
+
+* Host Operating System: Ubuntu 16.04 LTS
+* Build Container Operating System: Ubuntu 16.04.3 LTS
+* Kernel Version: 4.4.0-1049-aws
+* **Docker Server Version: 17.06.0-ce**
+* Storage Driver: aufs
+* Root Dir: /data/aufs
+* Backing Filesystem: extfs
+* Dirperm1 Supported: false
+* Cgroup Driver: cgroupfs
+* Shippable Official u16 Docker Images with TAG: `v6.6.2`
+
+## c7 sku
+* Host Operating System: CentOS 7
+* Build Container Operating System: CentOS 7
+* Kernel Version: 3.10.0-514.10.2.el7.x86_64
+* **Docker Server Version: 17.06.0-ce**
+* Storage Driver: overlay
+* Root Dir: /data
+* Backing Filesystem: xfs
+* Cgroup Driver: cgroupfs
+* Shippable Official c7 Docker Images with TAG: `v6.6.2`
+
+## w16 sku
+* Host Operating System: Windows Server 2016 Datacenter Edition (Version 1607)
+* Build Container Operating System: Windows Server Core 2016 v10.0.14393.1884
+* Docker Server Version: 17.10.0-ce
+* Shippable Official w16 Docker Images with TAG: `v6.6.2`
+
+## Shippable Official Docker Images
+
+These are the images used to run your CI jobs. The default image is picked up based on the `language` you set in your yml. All these images are available on our [Docker drydock Hub](https://hub.docker.com/u/drydock/). The source code is available in our [GitHub dry-dock org](https://github.com/dry-dock)
+
+If you would like to use your own CI images in place of the official images, instructions are [described here](/ci/custom-docker-image/)
+
+These are the official language images in this version:
+
+* [Common components](#common-644)
+* [Nodejs](#nod-644)
+* [Clojure](#clo-644)
+* [Go](#gol-644)
+* [PHP](#php-644)
+* [Java](#jav-644)
+* [Ruby](#rub-644)
+* [Python](#pyt-644)
+* [Scala](#sca-644)
+* [C/C++](#cpp-644)
+* [Dotnet](#dotnet-644)
+* [ASP.Net](#aspnet-644)
```

Comment:
All of these anchors are wrong. They should say `662` instead of `644`. This means all the anchors below also need to be updated.

Reply:
sure i will update it.